### DISCUSSION TITLE
Open Collective

### DISCUSSION THREAD

**Discussion by Simmonz** [redeclipse/base#1299] [General]
I found an Open Collective for Red Eclipse but I don't see any reference to this on the Red Eclipse website. Is this legit and something made by the Red Eclipse team ?

https://opencollective.com/redeclipse

Never mind, found it listed under the sponsor section.

**Comment by qreeves** [comment]
Yes, and the built-in GitHub sponsor stuff pays out to the OpenCollective as well.
